components/LinkGroup/src/index.scss

```diff
@@ -10,28 +10,32 @@
   margin-block-end: var(--denhaag-link-group-caption-margin-block);
 }
 
-.denhaag-link-group__navigation-list {
-  margin-block-start: var(--denhaag-link-group-navigation-list-margin-block-start);
-  margin-block-end: var(--denhaag-link-group-navigation-list-margin-block-end);
-  list-style: var(--denhaag-link-group-navigation-list-list-style);
-  padding-inline-start: var(--denhaag-link-group-navigation-list-padding-inline-start);
+.denhaag-link-group__list {
+  margin-block-start: var(--denhaag-link-group-list-margin-block-start);
+  margin-block-end: var(--denhaag-link-group-list-margin-block-end);
+  list-style: var(--denhaag-link-group-list-list-style);
+  padding-inline-start: var(--denhaag-link-group-list-padding-inline-start);
 
   @media (min-width: 1114px) {
     margin-block-start: var(--denhaag-space-block-sm);
   }
 }
 
-.denhaag-link-group__navigation-list-item {
-  padding-block-start: var(--denhaag-link-group-navigation-list-item-padding-block);
-  padding-block-end: var(--denhaag-link-group-navigation-list-item-padding-block);
+.denhaag-link-group__list-item {
+  padding-block-start: var(--denhaag-link-group-list-item-padding-block);
+  padding-block-end: var(--denhaag-link-group-list-item-padding-block);
 }
 
 .denhaag-link-group--dark .denhaag-link,
 .denhaag-link-group--dark .denhaag-link:hover,
-.denhaag-link-group--dark .denhaag-link:focus {
+.denhaag-link-group--dark .denhaag-link:focus-visible {
   color: var(--denhaag-link-group-dark-denhaag-link-color);
 }
 
+.denhaag-link-group--dark .denhaag-link:focus-visible {
+  outline-color: var(--denhaag-link-group-dark-denhaag-link-focus-outline-color);
+}
+
 .denhaag-link-group--dark {
   --denhaag-focus-border-color: var(--denhaag-link-group-dark-denhaag-link-focus-border-color);
```

Comment:
ik denk dat deze token `--denhaag-link-group-dark-denhaag-link-focus-border-color` nu door de rename in het token bestand niet meer bestaat, weet niet of hij nog nodig is, maar anders mag hij misschien hier ook weg?